modules/basicfuncs/list-funcs.c

```diff
@@ -24,6 +24,8 @@
 #include "scanner/list-scanner/list-scanner.h"
 #include "str-repr/encode.h"
 
+#include <pcre.h>
```

Comment:
Please use `compat/pcre.h` instead, that covers `PCRE_STUDY_JIT_COMPILE` and others.

Reply:
Thanks, done!